### ISSUE TITLE
Revealed Content is Not Keyboard Accessible.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#145]
Screengrab/videos (insert file names here)
2020 08 24, Revealed Content is Not Keyboard Accessible.mp4

Link
https://d8-stg.boston.gov/departments/emergency-medical-services/how-apply-ems-special-event-permit#in-person

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.5 > Chrome 84

Trusted Tester Test
n/a

Brief Description
When testing to see if form is accessible, tabbing occurs of background content, not on the form itself.

Steps to reproduce
1)	00:00, When video starts the Contact Us form is already shown, as preparation for creating a bug report.
2)	00:01, Pressing repeatedly on the tab button. 
3)	00:06, While pressing the tab button, the cursor is advancing the background content only. In other words, the tabbing is affecting the background content, not the revealed content.
a)	This is a bug as the tabbing should be on the revealed content.
b)	Note: Repeated test showing Contact form loaded for video and confirming tabbing is working on background content only.

Desired Result
•	Able to use the tabbing feature to move through the Contact Form for City of Boston.

Actual Result
•	Unable to use the tabbing feature to move through the Contact Form for City of Boston.
[Revealed Content is not Keyboard Assessible.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5120095/Revealed.Content.is.not.Keyboard.Assessible.mp4.zip)
